include/lsst/jointcal/AstromFit.h

```diff
@@ -67,13 +61,12 @@ private :
     int _LastNTrip; // last triplet count, used to speed up allocation
     double _referenceColor, _sigCol; // average and r.m.s color
     unsigned _nRefrac;
-    std::vector<double> _refracCoefficient; // fit parameter
+    double _refractionCoefficient; // fit parameter
```

Comment:
As it existed before, that vector only ever had length one. I'm guessing they were planning on expanding it later, but it made it very confusing, and resulted in a bunch of code elsewhere that actually did nothing (but it took a while to determine that). If we do need more than one in the future, we can implement it then.